docs/template_extension/new/op.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op.hpp"
+
+using namespace TemplateExtension;
+
+//! [op:ctor]
+Operation::Operation(const ngraph::Output<ngraph::Node>& arg, int64_t add) : Op({arg}), add(add) {
+    constructor_validate_and_infer_types();
+}
+//! [op:ctor]
+
+//! [op:validate]
+void Operation::validate_and_infer_types() {
```

Comment:
I think we need a more complex example operation to show how implement all overloaded methods

Reply:
It is a copy/past of existed operation